app/(articles)/[article].tsx

```diff
@@ -0,0 +1,96 @@
+import {View, Text, Pressable, StyleSheet, ScrollView} from 'react-native';
```

Comment:
Is this related to the PR? This looks like stuff from @ItsMeKimi? Concerned about merge conflict once they push their changes as well.